examples/platform/silabs/sl_systemview_config.h

```diff
@@ -0,0 +1,80 @@
+/***************************************************************************/ /**
```

Comment:
can you fix the comment banners 
the csa restyler doesn't like the silabs style at the end `*/ /**`

Reply:
done